pkg/entity/project/project.go

```diff
@@ -16,7 +18,10 @@ var client projects.ProjectsServiceClient
 var apiContext context.Context
 
 const (
-	descriptionFlag = "description"
+	descriptionFlag       = "description"
+	addMemberFlag         = "add-member"
+	removeMemberFlag      = "remove-member"
```

Comment:
perhaps the flag should be plural, since we can add multiple?

Reply:
good call